test/cog/chat/hipchat/template_processor_test.exs

```diff
@@ -107,4 +107,37 @@ defmodule Cog.Chat.HipChat.TemplateProcessorTest do
 
     assert expected == rendered
   end
+
+  test "handles link directives" do
+    directives = [
+      %{"name" => "link", "text" => "a link", "url" => "http://www.example.com"}
+    ]
+
+    rendered = TemplateProcessor.render(directives)
+    expected = "<a href='http://www.example.com'>a link</a>"
+
+    assert expected == rendered
+  end
+
+  test "handles link directive with a nil url" do
```

Comment:
How do these behave in chat? They look like links, but when you click, nothing happens? That sounds maddening.

I was thinking perhaps something along the lines of how we handle unrecognized directives:
https://github.com/operable/cog/blob/402bfba5f427642640a09ecf4c48291ecaf98164/lib/cog/chat/slack/template_processor.ex#L103-L106